src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php

```diff
@@ -0,0 +1,379 @@
+    }
+
+    /**
+     * @When I want to add a new country
+     */
+    public function iWantToAddANewCountry(): void
+    {
+        $this->client->buildCreateRequest();
+    }
+
+    /**
+     * @When I choose :countryName
+     */
+    public function iChoose(string $countryName): void
+    {
+        $this->client->addRequestData('code', $this->getCountryCodeByName($countryName));
+    }
+
+    /**
+     * @When I add it
+     */
+    public function iAddIt(): void
+    {
+        $this->client->create();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyCreated(): void
+    {
+        Assert::true(
+            $this->responseChecker->isCreationSuccessful($this->client->getLastResponse()),
+            'Country could not be created'
+        );
+    }
+
+    /**
+     * @Then the country :country should appear in the store
+     */
+    public function theCountryShouldAppearInTheStore(CountryInterface $country): void
+    {
+        Assert::true(
+            $this->responseChecker->hasItemWithValue($this->client->index(), 'code', $country->getCode()),
+            sprintf('There is no country with name "%s"', $country->getName())
+        );
+    }
+
+    /**
+     * @Then the country :country should have the :province province
+     * @Then /^(this country) should have the ("[^"]*" province)$/
+     */
+    public function theCountryShouldHaveTheProvince(CountryInterface $country, ProvinceInterface $province): void
+    {
+        Assert::true($this->responseChecker->hasItemWithValue(
+            $this->client->subResourceIndex('provinces', $country->getCode()),
+            'code',
+            $province->getCode()
+        ));
+    }
+
+    /**
+     * @Then the province should still be named :province in this country
+     */
+    public function theProvinceShouldStillBeNamedInThisCountry(ProvinceInterface $province): void
+    {
+        /** @var CountryInterface $country */
+        $country = $this->sharedStorage->get('country');
+        Assert::true($this->responseChecker->hasItemWithValue(
+            $this->client->subResourceIndex('provinces', $country->getCode()),
+            'code',
+            $province->getCode()
+        ));
+    }
+
+    /**
+     * @Then I should not be able to choose :countryName
+     */
+    public function iShouldNotBeAbleToChoose(string $countryName): void
+    {
+        $this->client->addRequestData('code', $this->getCountryCodeByName($countryName));
+        $response = $this->client->create();
+        Assert::false(
+            $this->responseChecker->isCreationSuccessful($response),
+            'Country has been created successfully, but it should not'
+        );
+        Assert::same($this->responseChecker->getError($response), 'code: Country ISO code must be unique.');
+    }
+
+    /**
+     * @When /^I want to edit (this country)$/
```

Comment:
It would be glorious for me if we have functions in this context (and all other contexts) sorted in the Given -> When -> Then order 🚀